apps/GroundControl/groundControl.py

```diff
@@ -1,11 +1,84 @@
+
+import serial
+
+MESSAGE_START = "<"
+MESSAGE_END = ">"
+ENCODING = "ascii"
+SERIAL_PORT = "COM8"
+BAUDRATE = 115200
+
+READY_MESSAGE = "<Arduino is ready>"
+
+#Used to control the ground vehicle
 class GroundControl():
-	#Used to control the ground vehicle
+    def __init__(self):
+        self.id = 0
+        self.open_serial()
+
+    # open serial port and wait for ready
+    def open_serial(self):
+        self.ser = serial.Serial(SERIAL_PORT, BAUDRATE)
+        while self.get_response() != READY_MESSAGE:
```

Comment:
Lets just put a print in here 
```suggestion
        print("Waiting for motor controller...")
```